bundle/API/Repository/TagsService.php

```diff
@@ -173,25 +173,40 @@ public function getTagSynonymCount(Tag $tag, array $languages = null, $useAlways
      * @param int $offset The start offset for paging
      * @param int $limit The number of content objects returned. If $limit = -1 all content objects starting at $offset are returned
      * @param bool $returnContentInfo
+     * @param array $sortClauses
+     * @param \eZ\Publish\API\Repository\Values\Content\Query\Criterion[] $additionalCriteria Additional criteria for filtering related content
      *
-     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException If the current user is not allowed to read tags
-     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException If the specified tag is not found
+     * @throws \eZ\Publish\Core\Base\Exceptions\UnauthorizedException
```

Comment:
Why the change in `@throws` tags here? Same goes for this change in `Core\TagsService` and `SignalSlot\TagsService` classes.

Reply:
It was probably due to PHP Storm complaining about never thrown exception. Fixed in . But to be honest, I don't see where does NotFoundException come from? SearchService doesn't throw it.

Fixed in https://github.com/netgen/TagsBundle/pull/98/commits/05f7dad27c86fbfdc247c7f08f2966bc63b3a183.

Reply:
This is a good catch indeed. `NotFoundException` is not thrown here, but I'd rather we fix it in one go later when cleaning up for v4.

Reply:
Fine with me :)